### DISCUSSION TITLE
目前，如果要让umi-ocr运行批处理任务更有效率，是优先考虑升级GPU还是CPU？

### DISCUSSION THREAD

**Discussion by lforlgg** [hiroi-sora/Umi-OCR#494] [Q&A]
看到有openGL选项，是不是意味着目前已经有用到GPU运算呢？
主要是靠cpu还是靠GPU？
渲染器选哪个更快，在什么情况下选哪一个为佳？
openGL ES
desk openGL
🙏

**Comment by hiroi-sora** [comment]
这个选项，是用显卡加速前端UI渲染，也就是让界面更流畅、更不占用CPU资源。一般来说 `OpenGL ES` 的兼容性最好，这也是默认项。对于少数显卡，如果界面出现乱码、显示不正确的问题，可以改为 `Desktop OpenGL` 或关闭硬件加速。

关于OCR的显卡加速问题，见 [FAQ](https://github.com/hiroi-sora/Umi-OCR/issues/447) 。

**Comment by lforlgg** [comment]
收到，多谢答复🌹
